src/methods/modular_programs/hard/q3/UniversityLife.java

```diff
@@ -0,0 +1,37 @@
+package methods.modular_programs.hard.q3;
+
+/*
+Write a program with a method called universityYear that asks the user for their undergraduate year level and prints
```

Comment:
Again, I feel like the question is unclear about where the scanner is asking for the data, and what the input to the method is.